pontoon/base/models.py

```diff
@@ -509,7 +509,13 @@ def get_top_instances(cls, qs):
             'most_suggestions': sorted(qs, key=lambda x: x.unreviewed_strings)[-1],
             'most_missing': sorted(
                 qs,
-                key=lambda x: x.total_strings - x.approved_strings - x.fuzzy_strings
+                key=lambda x: (
+                    x.total_strings -
```

Comment:
[nit] I wonder if that's possible to make the property called `missing` and use it here and in the template.

Reply:
That sounds like a good idea, I'll look into it.